src/test/java/racingGame/GameTest.java

```diff
@@ -0,0 +1,42 @@
+package racingGame;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class GameTest {
+
+    @Test
+    @DisplayName("자동차 수와 반복횟수를 매개변수로 게임 생성")
+    void create() {
+        Game game = new Game(3, 5);
+        assertThat(game.getCarsStatus().size()).isEqualTo(3);
```

Comment:
다음과 같이 해볼 수도 있을 것 같아요! 😄 
```suggestion
        assertThat(game.getCarsStatus()).hasSize(3);
```